Makefile

```diff
@@ -41,22 +41,16 @@ SRC_DIRS := cli pkg
 ###
 
 ALL_ARCH := amd64 arm arm64 ppc64le
+BASEIMAGE?=scratch
 
-# Set default base image dynamically for each arch
+# Image name ends with the -arch unless it's amd64
 ifeq ($(ARCH),amd64)
-    BASEIMAGE?=alpine
-endif
-ifeq ($(ARCH),arm)
-    BASEIMAGE?=armel/busybox
-endif
-ifeq ($(ARCH),arm64)
-    BASEIMAGE?=aarch64/busybox
-endif
-ifeq ($(ARCH),ppc64le)
-    BASEIMAGE?=ppc64le/busybox
+        IMARCH?=""
+else
+        IMARCH?='-$(ARCH)'
```

Comment:
whoops. I made a mistake in the tectonic-torcx makefile; these single quotes are incorrect.

Reply:
Ack, fixed to use double-quotes instead.